appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -243,13 +243,40 @@ public boolean ShowFilterBar() {
    * @param itemsList a YailList containing the strings to be added to the ListView
    */
   @SimpleProperty(description="List of text elements to show in the ListView.  This will " +
-                "signal an error if the elements are not text strings.",
-      category = PropertyCategory.BEHAVIOR)
+          "signal an error if the elements are not text strings.",
+          category = PropertyCategory.BEHAVIOR)
   public void Elements(YailList itemsList) {
     items = ElementsUtil.elements(itemsList, "Listview");
     setAdapterData();
   }
 
+//  @SimpleProperty(description="Query results to show in the ListView. "
+//          "It will show the text values for the first variable found in the bindings.",
+//          category = PropertyCategory.BEHAVIOR)
```

Comment:
Remove commented out code.

Reply:
A relic from the prior version that rather showed it as a property (which is more elegant IMO but then we cannot indicate the variable to be shown). Removed it